### ISSUE TITLE
Feature: Webhook Support

### ISSUE THREAD

**Issue by bdwyertech** [bdwyertech/better-cfn-signal#3]
Lets say I have a service (local or remote) with a healthcheck endpoint.

```bash
# Wait up to 10 minutes for a 200 response, retrying every 30s
better-cfn-signal --webhook http://127.0.0.1:8080/api/healthz --timeout 10m --retry 30s
```

This will block until it receives a 200, otherwise it will exit non-zero indicating timeout waiting for webook to be healthy.

**Comment by bdwyertech** [comment]
cc: @marcuslap

**Comment by bdwyertech** [comment]
resolved by #4
